### PR TITLE
Adding terraform scripts for privilege-grantor

### PR DESCRIPTION
**Pull request by abhimanyugupta07**
Terraform module for Privilege Grantor

### CODE REVIEW

File: privileges-grantor/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Overview
+
+Terraform module for setting up infrastructure for [Apiary Privilege Grantor](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-events/apiary-metastore-consumers/privileges-grantor).
```
Comment (barnharts4) [inline privileges-grantor/README.md]: Should be `[Apiary Privileges Grantor]` (plural) to match everywhere else.

File: privileges-grantor/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Example module invocation:
+```
+module "apiary-privilege-grantor" {
+  source = "git@github.com:HotelsDotCom/apiary-extensions-terraform.git/privileges-grantor"
+  pg_lambda_bucket  = "pg-s3-bucket"
+  pg_jars_s3_key    = "pg-s3-key"
+  pg_lambda_version = "4.1.0"
+  pg_metastore_uri  = "thrift://ip-address:9083"
+  metastore\_events\_sns\_topic = "arn:aws:sns:us-west-2:1234567:metastore-events-sns-topic"
```
Comment (barnharts4) [inline privileges-grantor/README.md]: The backslashes in this line are showing up in the rendered Markdown.

File: privileges-grantor/README.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+| security\_groups | Security groups in which Lambda will have access to. | list | n/a | yes |
+| subnets | Subnets in which Lambda will have access to. | list | n/a | yes |
+| tags | A map of tags to apply to resources. | map | `<map>` | no |
+
+## Usage
+
+Example module invocation:
+```
+module "apiary-privilege-grantor" {
+  source = "git@github.com:HotelsDotCom/apiary-extensions-terraform.git/privileges-grantor"
```
Comment (barnharts4) [inline privileges-grantor/README.md]: Shouldn't the github org be `ExpediaGroup`?

**Review by mroark1m — APPROVED** [review]
Looks good to me, will reapprove once the existing comments are all settled.

File: privileges-grantor/README.md (hunk 1 above)
Comment (abhimanyugupta07) [reply privileges-grantor/README.md]: Done.

File: privileges-grantor/README.md (hunk 2 above)
Comment (abhimanyugupta07) [reply privileges-grantor/README.md]: removed.

File: privileges-grantor/README.md (hunk 3 above)
Comment (abhimanyugupta07) [reply privileges-grantor/README.md]: Done.

**Review by max-jacobs — APPROVED** [review]
Looks good

**Review by barnharts4 — APPROVED** [review]
Looks good.

**Review by mroark1m — APPROVED** [review]

**Review by massdosage — APPROVED** [review]

**Pull request merged**